### ISSUE TITLE
Outlook message flags not read when NVDA started as admin

### ISSUE THREAD

**Issue by Novalis7747** [nvaccess/nvda#10629]
### Steps to reproduce:
- use a fresh NVDA 2019.3b1 portable copy
- run it as admin
- start outlook and navigate the message list of new mails
### Actual behavior:
- NVDA doesn't read any message flag like unread, forwarded, replyed and so on
### Expected behavior:
NVDA should read thos message flags
#### NVDA installed/portable/running from source:
NVDA portable 2019.3 Beta 1 (this bug was present in all Py3 alpha versions)
#### NVDA version:
2019.3b1 and all py3 alpha versions
#### Windows version:
Win 10 pro 2019 latest updates
#### Name and version of other software in use when reproducing the issue:
Microsoft Office 2013, 2016 365 and 365 plus Outlook
#### Other information about your system:
nothing special, I can reproduce this on 4 different devices and other users confirmed this bug as well
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
all up to NVDA 2019.2.1 worked well
attached NVDA logfiles and debug files for both scenarios, working and not working. I assume the bug will also occur when NVDA is fixed installed on the system since it will run in admin mode thene
[NVDA 2019.3b1_Outlook_message_flags_bug.zip](https://github.com/nvaccess/nvda/files/3964959/NVDA.2019.3b1_Outlook_message_flags_bug.zip)

**Comment by CyrilleB79** [comment]
Confirmed on my side with Outlook 2016 and installed version of NVDA 2019.3beta1.

To understand context, why do you need to launch NVDA with admin privileges? Does it have to do with Outlook or not?

Regarding technical analysis, I have not investigated. But maybe it is linked to the way NVDA gets this information. I think I recall that NVDA uses Outlook's COM interface to get this information. Probably it is not accessible with admin privileges.

Another test you can do: What does happen if you launch Outlook with admin privileges?

**Comment by Novalis7747** [comment]
Great that you could reproduce it with a installed 2019.3b1 .
You see, when you run the portable copy without admin rights several parts of Win10 are not accessible, an example is the device manager.
I testet to run the 2019.3b1 portable as admin and also started  Outlook with run as admin and guess what, the message flags are beeing read. Good hint 😉

Cheers,
  Ralf

**Comment by Adriani90** [comment]
So this means if NVDA is running as an administrator but the application you want to use is not, than NVDA shows also discrepancies. Is this the case with other applications as well? Or do you see this behavior only in Outlook?
cc: @michaelDCurran, @leonardder

**Comment by Novalis7747** [comment]
We saw that only in Outlook and the message flags under 2019.3.b1 and all earlier alphas. I would need to do more testing with other MS Office apps.
This is a showstopper for me because it happens on installed 2019.3.b1 as well (means not only on a portable version) and makes Outlook unusable.

**Comment by leonardder** [comment]
I've seen this in older versions of NVDA as well. It looks like this has to do with a user privilege conflict.

Anyway, running NVDA as admin is strongly discouraged. You should either install NVDA or live with the fact that some administrative tasks can't be done with a portable version of NVDA. I'm afraid there isn't that much we can do about this.

**Comment by lukaszgo1** [comment]
The problem here is, according to the other commends, that these flags are not read even if NVDA is installed. Perhaps this issue should be renamed?

**Comment by leonardder** [comment]
Correct, but also when run as admin, which shouldn't be done.

**Comment by Novalis7747** [comment]
When I install NVDA on a PC I have no choice to run it as an admin,  so again, even if NVDA is normal installed I see this problem on all of my PC’s and some other users saw it as well. Tell me what I have to do different to avoid this misbehavior and I’m happy.

**Comment by Novalis7747** [comment]
And again, that all works well with 2019.2.1 and all earlier versions of NVDA on all of my PCs, just with 2019.3 and earlier alphas I and other ppl can see this symptoms.

From: Ralf K.
Sent: Donnerstag, 19. Dezember 2019 22:01
To: nvaccess/nvda <reply@reply.github.com>; nvaccess/nvda <nvda@noreply.github.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [nvaccess/nvda] Outlook message flags not read when NVDA started as admin (#10629)

When I install NVDA on a PC I have no choice to run it as an admin,  so again, even if NVDA is normal installed I see this problem on all of my PC’s and some other users saw it as well. Tell me what I have to do different to avoid this misbehavior and I’m happy.

**Comment by leonardder** [comment]
> When I install NVDA on a PC I have no choice to run it as an admin, 

I don't understand this. When you install NVDA on a system, by default it is not run as admin at all. It is only run with UI Access, which makes it possible to get access to the interface of applications running as admin.

> so again, even if NVDA is normal installed I see this problem on all of my PC’s.

Are you saying that a bare install of NVDA without explicitly running it as admin has this problem as well?

I will investigate later today.

**Comment by leonardder** [comment]
On my end, a normally installed copy of NVDA without explicitly being started as admin, all works as normal. When I right click the desktop icon and run NVDA as admin, the flags no longer read, but this this known behavior for earlier versions as well.

**Comment by CyrilleB79** [comment]
On my end, I have performed the following tests and confirm what @leonardder  writes:
* NVDA run as user and Outlook run as user: OK, flags are read normally
* NVDA run as admin and Outlook run as user: NOK the flags are not read
I get this result either with NVDA 2019.2.1 portable or NVDA 2019.3beta1 installed. These tests were performed with Win10 1809 and Outlook 2016.

The real issue here is that you cannot access the device manager (and maybe other app) with NVDA run with normal privileges.
As @leonardder wrote, NVDA should not be run with admin privileges. I understand you do it to work-around another issue. But this should be discouraged and the root cause should be analysed and fixed if possible.
If the issue with Device manager cannot be fixed, it should be encouraged to run NVDA as admin only for such admin tasks.

**Comment by Novalis7747** [comment]
Confirmed at my end.  The other 2 users reporting this with an installed version of NVDA did run it as admin.
Since this is a normal behavior you can close this issue, I’m going to inform this users that running NVDA as admin is not recommended.
